lib/modules/manager/vendir/readme.md

```diff
@@ -1,30 +1,35 @@
-Renovate supports updating Helm Chart references and git references in vendir.yml via the [vendir](https://carvel.dev/vendir/) tool. Renovate requires the presence of a [vendir lock file](https://carvel.dev/vendir/docs/v0.40.x/vendir-lock-spec/) which is generated by vendir and should be stored in source code.
+Renovate can update Helm Chart references and Git references in `vendir.yml` files, with the [vendir](https://carvel.dev/vendir/) tool.
+
+For Renovate to work you must:
+
+- let vendir create a [vendir lockfile](https://carvel.dev/vendir/docs/v0.40.x/vendir-lock-spec/)
+- put the vendir lockfile in your repository
 
 ### Helm Charts
 
-It supports both https and oci helm chart repositories.
+Renovate supports HTTPS and OCI Helm chart repositories.
 
 ```yaml title="Example helm chart vendir.yml"
 apiVersion: vendir.k14s.io/v1alpha1
 kind: Config
 
 # one or more directories to manage with vendir
 directories:
-  - # path is relative to `vendir` CLI working directory
+  - # path is relative to the `vendir` CLI working directory
     path: config/_ytt_lib
     contents:
-      path: github.com/cloudfoundry/cf-k8s-networking
+      path: github.com/cloudfoundry/cf-k8s-networking # comment for review: this repo has been archived, what to do with the example?
```

Comment:
The `cloudfoundry/cf-k8s-networking` repository has been archived. What should we do with the config examples that link to that repository?

Reply:
Good idea to find a different example

Reply:
I don't know enough to find a good replacement for us. I created this issue to ask for help from the community:

- #29015

I'll remove the review comments, so that we can review and merge the other improvements. :smile: